docs/modules/ROOT/pages/how-tos/configuring/overriding-compute-resources.adoc

```diff
@@ -0,0 +1,87 @@
+= Overriding compute resources
+
+_For background on compute resources, see link:https://tekton.dev/docs/pipelines/compute-resources/[Compute Resources in Tekton]._
+
+{ProductName} tasks have default resource requests and limits.
+Some set them explicitly in the Task definition,
+others inherit them from the link:https://tekton.dev/docs/pipelines/compute-resources/#limitrange-support[LimitRange] in your namespace.
+
+If the default resources do not meet your needs, you can take advantage of Tekton's
+link:https://tekton.dev/docs/pipelines/pipelineruns/#specifying-taskrunspecs[taskRunSpecs] to override them.
+You can configure `taskRunSpecs` in the PipelineRun files in your `.tekton` directory. For example:
+
+[source,yaml]
+----
+kind: PipelineRun
+spec:
+  taskRunSpecs:
+    - pipelineTaskName: build-container
+      stepSpecs:
+        - name: build
+          computeResources:
+            requests:
+              memory: 10Gi
+            limits:
+              memory: 10Gi
+
+    - pipelineTaskName: prefetch-dependencies
+      computeResources:
+        requests:
+          cpu: '1'
+          memory: 2Gi
+        limits:
+          memory: 2Gi
+
+  pipelineSpec:
+    tasks:
+      # ...
+      - name: prefetch-dependencies
+        taskRef: ...
+      # ...
+      - name: build-container
+        taskRef:
+          resolver: bundles
+          params:
+            - name: kind
+              value: task
+            - name: name
+              value: buildah
+            - name: bundle
+              value: quay.io/konflux-ci/tekton-catalog/task-buildah:0.2@sha256:3f0913dfb85e9aeb9916e412d10329528ddf4c8fba9958cba5291ca8ee247f7e
```

Comment:
We will not be keeping this digest up to date in the documentation. Should we just use a tag or do you think that might result in confusion?

Reply:
Used just a version tag, that should be less confusing